src/proc_diskstats.c

```diff
@@ -358,6 +503,7 @@ int do_proc_diskstats(int update_every, unsigned long long dt) {
 			ddo_qops 	= config_get_boolean_ondemand(var_name, "queued operations", ddo_qops);
 			ddo_util 	= config_get_boolean_ondemand(var_name, "utilization percentage", ddo_util);
 			ddo_backlog = config_get_boolean_ondemand(var_name, "backlog", ddo_backlog);
+			ddo_space   = config_get_boolean_ondemand(var_name, "space", ddo_space);
```

Comment:
I think the default value should be:

```
ddo_space   = config_get_boolean_ondemand(var_name, "space", (ddo_space && mount_point)?1:0);
```

This way, the user should know that this setting is off, by reading the config.